x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackPlugin.java

```diff
@@ -102,6 +102,7 @@
                 public Void run() {
                     try {
                         Class.forName("com.unboundid.util.Debug");
+                        Class.forName("com.unboundid.ldap.sdk.LDAPConnectionOptions");
```

Comment:
I opened https://github.com/pingidentity/ldapsdk/issues/51 with the upstream library for this. Ideally we'll be able to remove this hack in the future.